pkg/cmdimpl/add.go

```diff
@@ -80,14 +87,20 @@ func updateParametersIfNecessary() error {
 	}
 
 	if params.Url == "" {
-		urlout, err := exec.Command("git", "remote", "get-url", "origin").CombinedOutput()
+		gitClient := git.New(nil)
+		repo, err := gitClient.Open(params.Dir)
+		if err != nil {
+			return err
+		}
+
+		remote, err := repo.Remote("origin")
 		if err != nil {
-			return wrapError(err, "could not get remote origin url")
+			return err
 		}
 
-		url := strings.TrimRight(string(urlout), "\n")
-		fmt.Printf("URL not specified; ")
-		params.Url = url
+		urls := remote.Config().URLs
+
+		params.Url = urls[0]
```

Comment:
We need to handle the case of empty urls here so the app does not panic but instead show a descriptive error message unless it is not possible to have empty urls after the err check on repo.Remote.